boards/arm/efr32_thunderboard/Kconfig.defconfig

```diff
@@ -57,3 +57,7 @@ endif # BT
 
 config REGULATOR
 	default y if SI7210
+
+config PWM_GECKO
+	default y
+	depends on PWM
```

Comment:
This seems to be redundant? `PWM_GECKO` is already being y-selected when the PWM node has the `okay` status.

Reply:
Sure! I didn't notice that it was already being selected.